catalog/api/app.py

```diff
@@ -796,6 +806,32 @@ async def workshop_post(request):
 
     raise web.HTTPConflict()
 
+@routes.get("/apis/babylon.gpte.redhat.com/v1/namespaces/{namespace}/catalogitems")
+@routes.get("/apis/babylon.gpte.redhat.com/v1/namespaces/{namespace}/catalogitems/{name}")
+async def openshift_api_proxy_with_cache(request):
+    namespace = request.match_info.get('namespace')
+
+    user = await get_proxy_user(request)
+    session = await get_user_session(request, user)
+
+    for catalog_namespace in session.get('catalogNamespaces', []):
+        if catalog_namespace['name'] == namespace:
+            break
+    else:
+        raise web.HTTPForbidden()
+
+    resp, cache_time = response_cache.get(request.path_qs, (None, None))
+    if resp != None and time() - cache_time < 60:
```

Comment:
should we make that value, 60, a parameter so we can adjust TTL more easily?

Reply:
Yeah, good call. I'll add that.